### DISCUSSION TITLE
Force Constants for SCPH calculation

### DISCUSSION THREAD

**Discussion by aowang-sjtu** [ttadano/alamode#30] [Q&A] [answered]
Dear Tadano
To determine the electron transport properties, the EPW package is often used to calculate the electron-phonon coupling with the QuantumESPRESSO force format. For some materials with unstable phonon modes , the SCPH calculations are needed. However, combining the EPW package with the ALM package is difficult. When we use the scph_to_qefc.py to create a new QE force constant file with anharmonic correction, this anharmonic correction is based on the supercell method implemented by ALM, which is different from QE with DFPT methods.This means the FCs should be exactly the same with both methods, which is difficult to achieve.  Is  there any way that we can convert ALM FC files into QuantumESPRESSO format, so that all calculations can be implemented with the supercell method?

**Comment by ttadano** [comment, marked as answer]
As long as the same computational codes/parameters are used, the harmonic FCs computed with the supercell method and those computed by DFPT with the q-mesh density commensurate with the supercell are very similar to each other. For example, harmonic FCs obtained with a 4x4x4 supercell are almost identical to those obtained by DFPT with a 4x4x4 q points. I have confirmed this for various systems.
(For this statement to be valid, all harmonic interactions within the supercell must be considered in ALM. Namely, you should set cutoff = 'None' for the harmonic part.)

**Reply by aowang-sjtu** [reply]
Ok, thank you very much!
